### ISSUE TITLE
Traj_XYZ.cpp compiling error with gcc v4

### ISSUE THREAD

**Issue by hainm** [Amber-MD/cpptraj#696]
hi,
This is the cpptraj's command in centos-6:
```
./configure --requires-flink -shared -openblas -noarpack -openmp -nosanderlib gnu
make libcpptraj -j4
```

And here is the error:
```
Traj_XYZ.cpp: In member function ‘Traj_XYZ::Type Traj_XYZ::DetermineFormat(std::string&, const std::string&) const’:
Traj_XYZ.cpp:38: error: ‘sscanf’ was not declared in this scope
Traj_XYZ.cpp: In member function ‘int Traj_XYZ::readXYZ(int, int, double*)’:
Traj_XYZ.cpp:189: error: ‘sscanf’ was not declared in this scope
make[1]: *** [Traj_XYZ.o] Error 1
make[1]: *** Waiting for unfinished jobs....
make[1]: Leaving directory `/work/cpptraj/src'
make: *** [libcpptraj] Error 2
```

**Comment by hainm** [comment]
Ah, I am using gnu v4.4

**Comment by drroe** [comment]
@hainm this should be addressed by #698 - let me know if not.

**Comment by hainm** [comment]
confirm. thanks.
